### PR TITLE
Github Actions

### PR DESCRIPTION
**Pull request by tannermiller-wf**
Move CI to Github Actions

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: .github/workflows/tests.yaml
Comment (connorszczepaniak-wk) [inline .github/workflows/tests.yaml:17]: Should we use a newer version of Go here? Or run a matrix of tests with a few versions of Go?

File: .github/CODEOWNERS
Comment (connorszczepaniak-wk) [inline .github/CODEOWNERS:1]: Based on the OSS guidelines, it sounds like the requirement is that we're required reviewers IFF the github actions themselves change.

I'm on board for just requiring us all the time too, though

Comment (connorszczepaniak-wk) [reply .github/CODEOWNERS:1]: Question though: does this require a review from us, or just request us as an optional reviewer?

**Review by connorszczepaniak-wk — APPROVED** [review]

**Comment by tannermiller-wf** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**
